apps/src/feedbackBlocks.js

```diff
@@ -67,23 +63,49 @@ var FeedbackBlocks = function(options, missingRequiredBlocks, missingRecommended
       level: options.level,
       blocks: this.xml
     }
-  });
+  };
+
+  this.div = document.createElement('div');
+  this.div.setAttribute('id', 'feedbackBlocksContainer');
+
   this.iframe = document.createElement('iframe');
   this.iframe.setAttribute('id', 'feedbackBlocks');
   this.iframe.setAttribute('allowtransparency', 'true');
+
   this.div.appendChild(this.iframe);
 };
 
 module.exports = FeedbackBlocks;
 
+/**
+ * Generate a url (with params) for the iFrame that's going to hold the
+ * blocks.
+ * TODO(elijah) replace this whole thing with an implementation that
+ * doesn't require an iframe, since we can now have multiple blocklys on
+ * the same page
+ */
+FeedbackBlocks.prototype.readonlyTemplateUrl_ = function () {
+  var params = {
+    app: this.iframeOptions.app,
+    js_locale: this.iframeOptions.options.locale,
+    locale_dir: this.iframeOptions.options.localeDirection
+  };
+
+  // Simple polyfill for $.params
+  var paramString = Object.keys(params).map(function (key) {
+    return key + "=" + encodeURIComponent(params[key]);
+  }).join("&");
+
+  return "/readonly_template?" + paramString;
```

Comment:
I know, I felt dirty just writing it. My thinking was that, since this is intended as a temporary fix, I wanted to avoid adding parameters to appOptions. The flipside is of course the ever-present danger that temporary additions can grow into permanent ones, but in this case I think it's justified.